Client/ApiClient.php

```diff
@@ -127,24 +128,33 @@ protected function getResponseData(
         Request $request
     ): ?string {
         $this->logger->debug(
-            "API Request:{$request->getMethod()} {$request->getUri()}",
+            "API Request",
             [
-                'body' => $request->getBody(),
+                'method' => $request->getMethod(),
+                'uri' => $request->getUri(),
+                'body' => (string)$request->getBody(),
             ]
         );
         try {
             $response = $this->client->sendRequest($request);
         } catch (HttpException $e) {
-            throw ApiRequestException::create((string) $request->getUri(), $e);
+            throw ApiRequestException::create((string)$request->getUri(), $e);
```

Comment:
Nope, check your coding style

Reply:
Looks like "symfony" coding style configuration from phpstorm is false... done!